tests/users.spec.js

```diff
@@ -0,0 +1,58 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const expect = chai.expect;
+const app = require('../app');
+const User = require('../models/user');
+
+const mongoose = require('mongoose');
+chai.use(chaiHttp);
+
+describe.only('[INDEX] GET /users/', () => {
+  it('If users not found,then array is empty', async () => {
+    const result = await chai.request(app).get('/users');
+    expect(result.header).to.have.property('content-type');
+    expect(result.header).to.have.property('content-type').contains('application/json');
+    expect(result.status).to.equal(200);
+    // expect(result.body).to.be.instanceof(Array);
```

Comment:
```suggestion
    expect(result.body).to.be.instanceof(Array);
```
decommenta questa riga , l'avrai dimenticata